2025-08-01 13:18:03.027 Uncaught app execution
Traceback (most recent call last):
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 58, in load
    dependencies, libzbar = load_objects(Path(''))
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 50, in load_objects
    deps = [
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 51, in <listcomp>
    cdll.LoadLibrary(str(directory.joinpath(dep)))
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\ctypes\__init__.py", line 452, in LoadLibrary
    return self._dlltype(name)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'libiconv.dll' (or one of its dependencies). Try using the full path with constructor syntax.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\streamlit\runtime\scriptrunner\exec_code.py", line 128, in exec_func_with_error_handling
    result = func()
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\streamlit\runtime\scriptrunner\script_runner.py", line 669, in code_to_exec
    exec(code, module.__dict__)  # noqa: S102
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_verifier_app.py", line 5, in <module>
    from pyzbar.pyzbar import decode
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\pyzbar.py", line 7, in <module>  
    from .wrapper import (
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\wrapper.py", line 151, in <module>
    zbar_version = zbar_function(
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\wrapper.py", line 148, in zbar_function
    return prototype((fname, load_libzbar()))
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\wrapper.py", line 127, in load_libzbar
    libzbar, dependencies = zbar_library.load()
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 60, in load
    dependencies, libzbar = load_objects(Path(__file__).parent)
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 54, in load_objects
    libzbar = cdll.LoadLibrary(str(directory.joinpath(fname)))
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\ctypes\__init__.py", line 452, in LoadLibrary
    return self._dlltype(name)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\libzbar-64.dll' (or one of its dependencies). Try using the full path with constructor syntax.
  Stopping...
(qr_env) PS C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis> ^C
(qr_env) PS C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis> streamlit run "QR code analysis.py"

  You can now view your Streamlit app in your browser.

  Local URL: http://localhost:8501
  Network URL: http://10.226.109.112:8501

2025-08-01 13:23:30.533 Uncaught app execution
Traceback (most recent call last):
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 58, in load
    dependencies, libzbar = load_objects(Path(''))
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 50, in load_objects
    deps = [
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 51, in <listcomp>
    cdll.LoadLibrary(str(directory.joinpath(dep)))
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\ctypes\__init__.py", line 452, in LoadLibrary
    return self._dlltype(name)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'libiconv.dll' (or one of its dependencies). Try using the full path with constructor syntax.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\streamlit\runtime\scriptrunner\exec_code.py", line 128, in exec_func_with_error_handling
    result = func()
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\streamlit\runtime\scriptrunner\script_runner.py", line 669, in code_to_exec
    exec(code, module.__dict__)  # noqa: S102
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\QR code analysis.py", line 4, in <module>
    from pyzbar.pyzbar import decode
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\pyzbar.py", line 7, in <module>  
    from .wrapper import (
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\wrapper.py", line 151, in <module>
    zbar_version = zbar_function(
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\wrapper.py", line 148, in zbar_function
    return prototype((fname, load_libzbar()))
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\wrapper.py", line 127, in load_libzbar
    libzbar, dependencies = zbar_library.load()
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 60, in load
    dependencies, libzbar = load_objects(Path(__file__).parent)
  File "C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\zbar_library.py", line 54, in load_objects
    libzbar = cdll.LoadLibrary(str(directory.joinpath(fname)))
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\ctypes\__init__.py", line 452, in LoadLibrary
    return self._dlltype(name)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.3056.0_x64__qbz5n2kfra8p0\lib\ctypes\__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\AChava05\OneDrive - Kenvue\AChava05\video_analysis\qr_env\lib\site-packages\pyzbar\libzbar-64.dll' (or one of its dependencies). Try using the full path with constructor syntax.
